src/Liip/RMT/Command/ReleaseCommand.php

```diff
@@ -136,13 +136,20 @@ protected function execute(InputInterface $input, OutputInterface $output)
         );
         Context::getInstance()->setParameter('new-version', $newVersion);
 
+        foreach (array('prerequisites', 'pre-release-actions', 'post-release-actions') as $listName) {
+            foreach(Context::getInstance()->getList($listName) as $action) {
+                $action->validateContext();
+            }
+        }
+        Context::get('version-persister')->validateContext();
```

Comment:
As discussed, because the generator work is a bit different than the other actions and the persister and there's no need at the time.
Adding it once we need it won't be a big issue.

Can we merge ? please, pretty please, PLEASE ! PLEASE !